go.mod

```diff
@@ -6,6 +6,7 @@ require (
 	github.com/gogo/protobuf v1.3.2
 	github.com/golang/mock v1.7.0-rc.1
 	github.com/golang/protobuf v1.5.3
+	github.com/josephspurrier/goversioninfo v1.4.0
```

Comment:
This file is read by prod users of rules_go, so we wouldn't want to place test deps here. 

Since the setup also requires binary files that don't seem that easy to regenerate *and* the actual change is pretty mechanical, in this case I would actually prefer *not* having the test. Sorry for the back and forth, but with xz in the meantime, we try to avoid this even more.